### ISSUE TITLE
[USER STORY] Links from results to BRIS and national registry

### ISSUE THREAD

**Issue by sskagemo** [STIRData/user-stories#20]
As a: User interested in facts about a specific company

I wish to: Be able to visit the authoritative source for information about the business, i.e. the relevant National Business Register or BRIS

So that I: Can verify that there are no recent changes to the data and look for additional information that might not be available from the STIRData platform (i.e. not covered by the STIRData Specification), for currently number of employees.

Related to epic #4 

Wireframe: [1](https://projects.invisionapp.com/share/JB11W6Z95C29#/screens/459935610)

Background:
For each individual company, it would be nice to be able to have links to the same company in both BRIS and the relevant national portal (if such service exists). This would be human readable versions.

If possible, it would be good to be able to download the data about the company as structured data directly from the STIRData portal (as well as the whole dataset from a query, ref #14 ), but if there exists an national api, it would be good to be able to go directly to that also.

In other words, for each entry, a set of links (with icons?) for BRIS, national portal (human readable), download, national portal (machine readable).

**Comment by sskagemo** [comment]
Screenshot from BRIS. I also realised that BRIS has not made it easy to link to individual entries (it is a search and with a captcha ... so that might not be possible).
https://e-justice.europa.eu/489/EN/business_registers__search_for_a_company_in_the_eu

![image](https://user-images.githubusercontent.com/5012227/122586157-975daf00-d05c-11eb-819e-c7fe8cdc685e.png)

**Comment by jakubklimek** [comment]
Yes, linking to BRIS is not possible due to how it is made - ID is not a param in URI, there is captcha, etc.

Similarly, link to human readable webpage about a company in official Czech BR cannot be generated from the data - it is available only via web search form

**Comment by tzortzispanagiotis** [comment]
@sskagemo @jakubklimek This can be supported in this [wireframe](https://projects.invisionapp.com/share/JB11W6Z95C29#/screens/459935610) (instead of the fact that direct linking, as you described, is not possible)
